networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -26,4 +26,38 @@ The Network Observability Operator is now stable and the release channel is upgr
+
+[id="network-observability-operator-1.2.0-bug-fixes"]
+=== Bug fixes
+
+* Previously, after changing the `namespace` value in the FlowCollector spec, eBPF Agent pods running in the previous namespace were not appropriately deleted. Now, the pods running in the previous namespace are appropriately deleted. (link:https://issues.redhat.com/browse/NETOBSERV-774[NETOBSERV-774])
+
+* Previously, after changing the `caCert.name` value in the FlowCollector spec (such as in Loki section), FlowLogs-Pipeline pods and Console plugin pods were not restarted, therefore they were unaware of the configuration change. Now, the pods are restarted, so they get the configuration change. (link:https://issues.redhat.com/browse/NETOBSERV-772[NETOBSERV-772])
+
+* Previously, network flows between pods running on different nodes were sometimes not correctly identified as being duplicates because they are captured by different network interfaces. This resulted in over-estimated metrics displayed in the console plugin. Now, flows are correctly identified as duplicates, and the console plugin displays accurate metrics. (link:https://issues.redhat.com/browse/NETOBSERV-755[NETOBSERV-755])
+
+* Previously, the "reporter" option in the console plugin, which is used to filter flows based on the observation point of either source node or destination node, mixed the flows regardless of the node observation point. This was due to network flows being incorrectly reported as Ingress or Egress at the node level.  Now, the network flow direction reporting is corrected. The "reporter" option filters for source observation point, or destination observation point, as expected. (link:https://issues.redhat.com/browse/NETOBSERV-696[NETOBSERV-696])
+
+* Previously, for agents configured to send flows directly to the processor as GRPC+protobuf requests, the submitted payload could be too large and is rejected by the processors' GRPC server. This occured under  very-high-load scenarios and with only some configurations of the agent. The agent logged an error message, such as: _grpc: received message larger than max_. As a consequence, there was information loss about those flows.
+Now, the GRPC payload is split into several messages when the size exceeds a threshold. As a result, the connectivity is maintained. (link:https://issues.redhat.com/browse/NETOBSERV-617[NETOBSERV-617])
+
+* Previously, you could install the Network Observability Operator using a custom namespace. Now, the Operator must be installed in the `openshift-netobserv-operator` namespace. You cannot automatically upgrade to the new Operator version if you previously installed the Network Observability Operator using a custom namespace. If you previously installed the Operator using a cusom namespace, you must delete the instance of the Operator that was installed and re-install your operator in the `openshift-netobserv-operator` namespace. (link:https://issues.redhat.com/browse/NETOBSERV-907[NETOBSERV-907])
```

Comment:
Not sure if we want to include something here about why, and call out the new metric collection via Prometheus

Reply:
I guess, the only "why" i can gather is because we added the conversion webhook and now can't have installMode OwnNamespace (and I only gather this from comments here https://issues.redhat.com/browse/NETOBSERV-907). I think I need some help connecting the dots about metric collection and Prometeus and the conversion webhook. @msherif1234 do you think this bug release note needs more context?

Reply:
> I guess, the only "why" i can gather is because we added the conversion webhook and now can't have installMode OwnNamespace (and I only gather this from comments here https://issues.redhat.com/browse/NETOBSERV-907). I think I need some help connecting the dots about metric collection and Prometeus and the conversion webhook. @msherif1234 do you think this bug release note needs more context?

- 1st there is typo `cusom` ->`custom`
- Now why we need custom namespace
for the metric part we have to add specific annotation to the namespace where we deploy the operator in ` openshift.io/cluster-monitoring=true`, and we can't do that for namespace that was shared by other operators `openshift-operators` hence we have to create our own.
- Why operator can't be installed in any namespace as it used to be 
in this release we introduced `conversion webhook` which forces us to change CSV based on  
https://olm.operatorframework.io/docs/advanced-tasks/adding-admission-and-conversion-webhooks/#conversion-webhook-rules-requirements
the side effect of this change is we no longer sees all avaliable namespace lists

Reply:
Okay, last week I ammended this release note. Hopefully its more accurate now. I spoke with Joel, and the two bugs about namespace changes were related (being limited to a single namespace, and pinning a name for it).  For metrics we had to choose a single namespace name, because metrics wouldn't be collected if the user installed in a different namespace than the suggested one. So it is okay to keep these two bug release notes together. I will just add the second bug # to the release note now.